[2025-04-27T21:55:05.452+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-04-27T21:50:00+00:00 [queued]>
[2025-04-27T21:55:05.461+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-04-27T21:50:00+00:00 [queued]>
[2025-04-27T21:55:05.461+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-27T21:55:05.476+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): load_mysql_data> on 2025-04-27 21:50:00+00:00
[2025-04-27T21:55:05.481+0000] {standard_task_runner.py:60} INFO - Started process 4412 to run task
[2025-04-27T21:55:05.485+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'mymind_mongo_to_mysql_etl', 'load_mysql_data', 'scheduled__2025-04-27T21:50:00+00:00', '--job-id', '3760', '--raw', '--subdir', 'DAGS_FOLDER/mymind_etl_dag.py', '--cfg-path', '/tmp/tmp8b20pv_s']
[2025-04-27T21:55:05.486+0000] {standard_task_runner.py:88} INFO - Job 3760: Subtask load_mysql_data
[2025-04-27T21:55:05.532+0000] {task_command.py:423} INFO - Running <TaskInstance: mymind_mongo_to_mysql_etl.load_mysql_data scheduled__2025-04-27T21:50:00+00:00 [running]> on host 3ea385d6ee71
[2025-04-27T21:55:05.629+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='mymind_mongo_to_mysql_etl' AIRFLOW_CTX_TASK_ID='load_mysql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-27T21:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-27T21:50:00+00:00'
[2025-04-27T21:55:05.631+0000] {mymind_etl_dag.py:284} INFO - Iniciando carga a MySQL: 3 usuarios, 7 transcripciones.
[2025-04-27T21:55:05.631+0000] {mymind_etl_dag.py:295} INFO - Cargando 3 usuarios en la tabla users...
[2025-04-27T21:55:05.640+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-04-27T21:55:05.655+0000] {sql.py:547} INFO - Done loading. Loaded a total of 3 rows into users
[2025-04-27T21:55:05.656+0000] {mymind_etl_dag.py:303} INFO - Usuarios cargados exitosamente.
[2025-04-27T21:55:05.656+0000] {mymind_etl_dag.py:322} INFO - Cargando 7 transcripciones en la tabla transcriptions...
[2025-04-27T21:55:05.663+0000] {base.py:83} INFO - Using connection ID 'mysql_mymind_dw' for task execution.
[2025-04-27T21:55:05.670+0000] {sql.py:547} INFO - Done loading. Loaded a total of 7 rows into transcriptions
[2025-04-27T21:55:05.671+0000] {mymind_etl_dag.py:330} INFO - Transcripciones cargadas exitosamente.
[2025-04-27T21:55:05.671+0000] {mymind_etl_dag.py:335} INFO - Carga a MySQL completada con Ã©xito.
[2025-04-27T21:55:05.671+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-27T21:55:05.682+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=mymind_mongo_to_mysql_etl, task_id=load_mysql_data, execution_date=20250427T215000, start_date=20250427T215505, end_date=20250427T215505
[2025-04-27T21:55:05.736+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-27T21:55:05.766+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
